include/ocpp/v201/charge_point.hpp

```diff
@@ -257,6 +257,15 @@ class ChargePointInterface {
     virtual void on_external_limits_changed(const std::variant<ConstantChargingLimit, ChargingSchedule>& limit,
                                             double percentage_delta, ChargingLimitSourceEnum source) = 0;
 
+    /// \brief Notifies the ChargePoint that an external limit has been cleared.
+    /// This shall send a ClearedChargingLimitRequest when called.
+    // This may send TransactionEventRequest if the \p percentage_delta is greater than our LimitChangeSignificance.
+    /// \param percentage_delta the percent changed from the existing limits
+    /// \param source the source of the external limit
+    /// \param evse_id if provided checks for transactions on the provided evse
+    virtual void on_external_limit_cleared(double percentage_delta, ChargingLimitSourceEnum source,
+                                           std::optional<int32_t> evse_id) = 0;
```

Comment:
General comment - I put `evse_id` first in my update, you put it last. Which way do we want to normalize it?

Reply:
@folkengine made a good point that required parameters should probably go first when we were pairing. Since evse_id is optional I think putting it at the end makes sense.